### DISCUSSION TITLE
Synology chat notification from synoai

### DISCUSSION THREAD

**Discussion by JoeyJoe6** [djdd87/SynoAI#209] [Q&A]
Here is a general guide I was following: https://kb.synology.com/en-us/Surveillance/tutorial/how_do_I_send_SS_data_chat_via_webhooks
I thought I could utilize synoai to send notification via webhook into synology chat. 

I populated the notifications with:
  "Notifiers": [
	{
    "Type": "Webhook",
    "URL": "http://192.168.xxx.xxx:5000/webapi/entry.cgi?api=SYNO.Chat.External&method=\"incoming\"&version=2&token=xxx",
    "Method": "POST",
	"SendImage": "true",

The synoai logs note success for a response, but I believe I am missing more info for the json body aka payload. 

Any suggestions?  thanks!!

**Comment by djdd87** [comment]
Unfortunately you need to send the message in a particular JSON format, which the standard notification behaviour will not do. I had partially started implementing this before the rewrite which I'm still in the middle of (time is very limited at the moment).
